staging/src/k8s.io/legacy-cloud-providers/OWNERS

```diff
@@ -5,12 +5,12 @@ approvers:
 - dims
 - andrewsykim
 - cheftako
-- mcrute
```

Comment:
cc @mcrute https://github.com/kubernetes/kubernetes/commits?author=mcrute

Reply:
I haven't had much time to dedicate to this lately so I'm okay with being removed as an active maintainer. Any reason not to just push me into emeritus instead?

Reply:
I did move you to `emeritus_approvers`, I think the way the diff is rendered makes it difficult to see that :P

Reply:
Oh, read the diff out of context. I hope to have more time in the future to contribute to cloud provider again. Until then...

/lgtm